### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by glennybob** [GerryDazoo/Slinger#599] [General]
Gerry - I want to thank you for your time to come up with this program. It takes what would be a useless box and makes it works well. In addition, your ongoing support is great!

Is there a way to send you a contribution for your hard work?

Thanks again!

**Comment by lead2crm** [comment]
Agreed, while my box isnt working at the moment, it was a sublime 2-days. I'll continue to debug.

But really, thank you so much for building this solution, it really made my day when I saw it went open source. I always suspected that we never needed SlingBox to host this, at all. I'm excited to see what other custom solution come in the future, like a web app player with integrated controls.

**Comment by GerryDazoo** [comment]
Thanks for the kind words. I’m glad it’s working for you. No need for any contributions but we could discuss naming rights for your first born. 😊

 

-Gerry

 

From: Glenn McDowell ***@***.***> 
Sent: Monday, May 15, 2023 3:44 PM
To: GerryDazoo/Slinger ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GerryDazoo/Slinger] Appreciation (Discussion #599)

 

Gerry - I want to thank you for your time to come up with this program. It takes what would be a useless box and makes it works well. In addition, your ongoing support is great!

Is there a way to send you a contribution for your hard work?

Thanks again!

**Reply by glennybob** [reply]
Too late - she is in her 40's!

**Comment by highfivin** [comment]
It's kind of a coincidence that this thread was created 2 days ago. 

I was going to leave comments expressing my appreciation of the work you have done here. I can't thank you enough

I set this up just over a month ago on my raspberry pi and it has been working flawlessly. I created a service to auto-start the server

I now have all 3 of my boxes working perfectly. 
SB500, M1 and M2 

I have customized the base DirecTv remotes to a more functional and visually pleasing layout
2 different version were created for the HR54/44 and the HR24 DVRs

Once I figured out that the SB returns all supported codes for the current configuration, I created a test remote with all the codes and recorded what each code controlled. I then created new remotes for mobile and web with a dark themed background 

![HR54-remote](https://github.com/GerryDazoo/Slinger/assets/133918667/a964f3e1-3208-43fe-af7b-4a4856041e13)

**Reply by highfivin** [reply]
This looks perfect on the mobile on a single screen with no scrolling needed, even though it appears very large here
